docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -89,6 +89,7 @@ property key | Description | Default |
 `osinfo.ipv4_list_size`| Limit the length of the ipv4 list size. |`10`|
 `collector.grpc_channel_check_interval`|grpc channel status check interval.|`30`|
 `collector.heartbeat_period`|agent heartbeat report period. Unit, second.|`30`|
+`collector.times_send_properties`|How many times do report a properties to the backend, Each time interval is {@link Collector#HEARTBEAT_PERIOD} |`10`|
```

Comment:
It's hard to express this config item properly, but this statement is abs. not what you really mean, according to the logic, it should be

```suggestion
`collector.times_send_properties`|The agent sends the instance properties to the backend every `collector.heartbeat_period * collector.times_send_properties` seconds |`10`|
```

Reply:
I prefer `collector.times_send_properties` to `collector.properties_report_period_factor`